package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "tidepool-uploader",
-  "version": "2.30.1",
+  "version": "2.30.0-i-sens-hid.1",
```

Comment:
Same as version query above.

Reply:
You know what - since I'm already making changes, I'll change this to `v2.30.1-i-send-hid.1` to make it clearer that I'm only adding a new suffix.